### ISSUE TITLE
Thumbnail view using poster instead of backdrop

### ISSUE THREAD

**Issue by PenDrag0n7** [jellyfin/jellyfin-androidtv#3092]
### Describe the bug

Thumbnail view should automatically switch to using backdrop image instead of posters, same way Web app does it.

### Application version

v0.15.12

### Where did you install the app from?

None

### Device information

Nvidia Shield Pro

### Android version

12

### Jellyfin server version

10.8.10

**Comment by nielsvanvelzen** [comment]
This may already be fixed in the 0.16 beta.

**Comment by PenDrag0n7** [comment]
Nope, just tested latest beta 0.16 beta 4, it still using poster images in thumnail view.

**Comment by PenDrag0n6989** [comment]
any plans to fix this
